.github/workflows/notebook-weekly-test.yml

```diff
@@ -0,0 +1,81 @@
+# This code is a Qiskit project.
+#
+# (C) Copyright IBM 2024.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+name: Test notebooks that submit jobs
+on:
+  schedule:
+    - cron: "0 0 * * 1" # At 00:00 on Monday
+  workflow_dispatch:
+jobs:
+  execute:
+    name: Execute notebooks
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
+          cache: "pip"
+
+      - name: Install Linux dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install texlive-pictures texlive-latex-extra poppler-utils graphviz
+
+      - name: Install Python packages
+        # This is to save our account in the next step. Note that the
+        # package will be re-installed during the "Run tox" step.
+        run: pip install qiskit-ibm-runtime tox
+
+      - name: Save IBM Quantum account
```

Comment:
Thoughts on DRYing this and preceding steps by creating a custom action locally? https://docs.github.com/en/actions/creating-actions

Not a deal breaker, but might be interesting to learn and helps keep the code maintainable.

Reply:
Good call, done in 0bc5f57 and I think it's much neater. Makes the individual actions smaller too.